src/Midle.js

```diff
@@ -2,61 +2,88 @@ import React from "react";
 import Twit from "./Twit";
 import style from "./pizza.module.css";
 import Pizzabox from "./Pizzabox";
-import italian from "./italian.jpg";
-import veg from "./veg.jpg";
-import paneer from "./paneer.jpg";
-import masroom from "./masroom.jpg";
-import pasta from "./pasta.jpg";
-import mexican from "./mexican.jpg";
+import * as Al from "./assets/images";
 
 const Midle = () => {
   const pizza = [
```

Comment:
aa array ne tu biji file ma lakhi shake 6 and ahiya import karavi ne use kari le

Reply:
resolved